test/UID2.Client.Test/KeyParserTests.cs

```diff
@@ -116,5 +116,66 @@ public void ParseErrorKeyList()
             Assert.ThrowsAny<Exception>(() => KeyParser.Parse(@"{""body"": [{""id"": ""test""}]}"));
             Assert.ThrowsAny<Exception>(() => KeyParser.Parse(@"{""body"": [{""id"": 5}]}"));
         }
+
```

Comment:
in the new tests below, could we populate the Keys section with some legit keys and then test with missing/empty/legit site data and still do  `Assert.True(keyContainer.TryGetKey(X, out key));` assertions to verify that keys still parsed correctly?

Reply:
Done